helm/dagster/schema/schema_tests/test_workspace.py

```diff
@@ -119,3 +119,26 @@ def test_workspace_renders_from_helm_dagit(template: HelmTemplate):
         assert grpc_server["grpc_server"]["host"] == server.host
         assert grpc_server["grpc_server"]["port"] == server.port
         assert grpc_server["grpc_server"]["location_name"] == server.host
+
+
+def test_workspace_renders_empty(template: HelmTemplate):
+    servers = []
```

Comment:
mypy is complaining about this line. Could you add this type annotation?

```suggestion
    servers: List[Server] = []
```